GestureHandler.js

```diff
@@ -109,11 +109,13 @@ const stateToPropMappings = {
   [State.END]: 'onEnded',
 };
 
-function canUseNativeParam(param) {
+function isEventHandler(param, name) {
```

Comment:
but this check if it is *not* an event handler. Also we do `param !== undefined` which will filter some other things too. I suggest renaming to `isConfigParam`